pagopa/io-app
# fix: [IOBP-128] Fix bottom sheet padding regression

## Short description
This PR fixes a regression in the bottom sheet modal visualization

## List of changes proposed in this pull request
- `ts/utils/hooks/bottomSheet.tsx`: fixed bottom sheet bottom padding regression
- `ts/features/design-system/core/DSBottomSheet.tsx`: added a new type of bottom sheet to test. This bottom sheet has a very long content and its snap point should snap below the upper safe-area limit
- `ts/features/idpay/initiative/timeline/components/TimelineDetailsBottomSheet.tsx`: replaced legacy bottom sheet

## How to test
Navigate to the Bottom Sheet screen in the DS section and make sure all types of modals are displayed correctly.
This fix should be tested on both iOS and Android.

## Acknowledgement

@SabPicariello
